.github/dependabot.yml

```diff
@@ -9,6 +9,7 @@ updates:
       timezone: Europe/Berlin
     ignore:
       - dependency-name: devtools-protocol
+      - dependency-name: chromium-bidi
```

Comment:
Dependabot always messes up the `package-lock.json` so disabling it.

Reply:
Lets split into 2 different PRs, as if one of the changes (roll Mapper + update cache) is failing, the other will have to be rolled back as well

Reply:
Done will send a PR for the chromium-bidi bump separately.